README.md

```diff
@@ -23,3 +23,4 @@ Contributions, issues, and feature requests are welcome!
 ## Show your support
 
 Give a ⭐️ if you like this project!
+
```

Comment:
- [ ] **Required:** The link to your screenshot is broken, let's fix it to have a visual of what the website is, right❓